src/typescript/azure-openapi-validator/rules/utilities/rules-helper.ts

```diff
@@ -39,4 +39,9 @@ function getResponseSchema(response: Object, doc): any {
     return schemaProperties;
   }
   return schema.properties;
+}
+
+export function IsValidOperation(operation: string): boolean {
```

Comment:
IsValidOperation => isValidOperation. Function naming the first  char lowercase.

Reply:
it just make sure the rule don't crash , there are other rules to check this